conjurapi/router_v5.go

```diff
@@ -50,11 +50,47 @@ func (r RouterV5) CheckPermissionRequest(resourceID, privilege string) (*http.Re
 	)
 }
 
+func (r RouterV5) ResourceRequest(resourceID string) (*http.Request, error) {
+	account, kind, id, err := parseID(resourceID)
+	if err != nil {
+		return nil, err
+	}
+
+	requestURL := makeRouterURL(r.resourcesURL(account), kind, url.QueryEscape(id))
+
+	return http.NewRequest(
+		"GET",
+		requestURL.String(),
+		nil,
+	)
+}
+
+func (r RouterV5) ResourcesRequest(filter *ResourceFilter) (*http.Request, error) {
+
+	var query []string
+	if filter != nil {
+		if filter.Kind != "" {
```

Comment:
These two conditions should probably be joined on a single line.

Reply:
I wrote it this way because there are going to be additional filters that will need to be appended to the query.